### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by jspricke**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -7,13 +7,11 @@ on:
...
 jobs:
   build_noetic:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
-      - uses: HemaZ/ros-deb-builder-action@main
+      - uses: jspricke/ros-deb-builder-action@main
         with:
-          DEB_DISTRO: noetic
+          DEB_DISTRO: focal
```
Comment (HemaZ) [inline .github/workflows/build.yml:14]: ah :man_facepalming: of course its not noetic

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -7,13 +7,11 @@ on:
 
 jobs:
   build_noetic:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```
Comment (HemaZ) [inline .github/workflows/build.yml:10]: This should stay 20.04 right? as ROS noetic binaries are only available on Ubuntu 20.04 focal?

Comment (jspricke) [reply .github/workflows/build.yml:10]: no, the Action host system runs only on 22.04, the build environment inside uses focal, as specified.

Comment (HemaZ) [reply .github/workflows/build.yml:10]: ooh, amazing. thanks for clarifying that.

**Review by ibraEssam — APPROVED** [review]
thanks @jspricke for looking at this. really appreciate it :tada:

**Pull request merged**